devops/test_configs.json

```
@@ -2,22 +2,22 @@
   "lts": [
     {
       "config": "lin_intel",
-      "name": "SYCL E2E on Intel CPU/GEN9 GPU",
+      "name": "SYCL E2E on Intel CPU/GEN12 GPU",
       "runs-on": [
         "Linux",
-        "gen9"
+        "gen12"
       ],
       "image": "${{ inputs.intel_drivers_image }}",
       "container_options": "-u 1001 --device=/dev/dri --privileged --cap-add SYS_ADMIN",
       "targets": "ext_oneapi_level_zero:gpu;opencl:gpu;opencl:cpu",
       "cmake_args": ""
     },
     {
-      "config": "l0_gen9",
-      "name": "L0 GEN9 LLVM Test Suite",
+      "config": "l0_gen12",
+      "name": "L0 GEN12 LLVM Test Suite",
```

```suggestion
      "name": "L0",
```
LLVM Test Suite?

What is this for? Don't `lin_intel` cover this testing or it validates only OpenCL backend?

---

IIRC, we use this one in post-commit. Again, I'm planning on dropping this entire config soon.